apps/redis/templates/services/app/dc.yml.j2

```diff
@@ -30,15 +30,15 @@ spec:
               command:
                 - "/bin/bash"
                 - "-c"
-                - "redis-cli ping"
+                - "'redis-cli ping && touch /data/healthcheck'"
```

Comment:
Should we delete it after to make sure it is created each time?

Reply:
I don't think so because you cannot touch something if you don't have the permission to touch it :sweat_smile: (I mean write permission, because `touch` updates touched file metadata). In other words, if the file already exists in a read-only volume or a volume that is no longer mounted, you will get an error; and this is what we expect.